.travis.yml

```diff
@@ -19,8 +19,9 @@ install:
   - pip install coverage
 
 script:
-  - flake8 anybox 
-  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox 
+  - flake8 anybox
+  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox
```

Comment:
@bwrsandman to use the same commands as those specified in the documentation http://docs.anybox.fr/anybox.recipe.odoo/current/contributing.html#launching-static-analysis-and-unit-tests